docs/monitoring/reports-and-components.md

```diff
@@ -47,10 +47,14 @@ Each alert query is visible in the Alert Rules list.
 
 By checking out the [Integrations documentation](https://sematext.com/docs/integration/) you can learn more about how to monitor various programming languages, operating systems, containers (Docker, Kubernetes, Mesos), Cloud IaaS / PaaS services, iOS, Android, AWS EC2, AWS S3, Log Management, and log shippers. There you will find detailed information on how to install and configure different agents required to start sending your logs and metrics data to Sematext Cloud.
 
-### Groups
-Report components can be grouped by tags using the `Groups` drop down menu in the Monitoring Report Menu. One of the [available tags](https://sematext.com/docs/tags/common-schema/) for grouping is the `os.kernel` tag, which will put hosts running the same Linux kernel in the same group. This can be useful to check whether some hosts need to be upgraded with a newer kernel.
+### Grouping
+Data in report components can be grouped by tags using the `Groups` drop-down menu in the Monitoring Report Menu. For example, one of the [available tags](https://sematext.com/docs/tags/common-schema/) for grouping is the `os.kernel` tag, which will put hosts running the same Linux kernel in the same group. This can be useful to check whether some hosts need to be upgraded with a newer kernel.
```

Comment:
> This can be useful to check whether some hosts need to be upgraded with a newer kernel.

In the Inventory reports, yes.  But here (looking at the screenshot) this wouldn't apply.

Reply:
> In the Inventory reports, yes. But here (looking at the screenshot) this wouldn't apply.

Why not? It's convenient to group by kernel version in the Hosts Overview report and check for hosts running outdated or custom kernels, etc.

Reply:
Because what I think we are showing on the screenshot I saw here yesterday is a timeseries chart where we showed some performance metric grouped by kernel.  Yes, this will let you see that you have 1 or more different kernels, but the primary goal of timeseries charts with metrics is to see performance.  If the primary interest is knowing which kernels are being used in your infra than the ideal place to go is Infra > Inventory > Servers.  That was my reasoning.